code/modules/species/outsider/vox.dm

```diff
@@ -123,11 +123,12 @@
 
 /datum/species/vox/equip_survival_gear(var/mob/living/carbon/human/H)
 	H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/vox(H), slot_wear_mask)
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(H),slot_shoes)
 
-	if(istype(H.get_equipped_item(slot_back), /obj/item/storage/backpack))
-		H.equip_to_slot_or_del(new /obj/item/tank/nitrogen(H), slot_r_hand)
+	if(istype(H.get_equipped_item(slot_back), /obj/item/storage/backpack)) // This is mostly for station Vox
+		H.equip_to_slot_or_del(new /obj/item/tank/emergency/nitrogen/double(H), slot_belt)
```

Comment:
The issue is that you can't hold a regular tank in any slot, except for the back.
This is especially an issue when you have a backpack(on the ship/station), because if the tank is placed in your hand - you just instantly drop it due to being put in cryogenic storage.

Reply:
And if this matters, Vox already filters air with their gasmasks, so the tank isn't really needed.

Reply:
If this is the case, I think it is worth considering making tanks equippable on the belt slot. I know Vox are not looked on favorably but I don't think it's a good look to give them tanks reserved for emergency decompression. @Cupax3 thoughts?

Reply:
> And if this matters, Vox already filters air with their gasmasks, so the tank isn't really needed.

Oh. Are you certain the partial pressure is good? If so, the tank isn't really needed at all.

Reply:
Walked around with a mask on, no tank - didn't suffocate.
I want to keep the tank in mainly for the fact Vox on off-ship spawn on a decompressed(?) map, as far as I know.

Reply:
Why not make those spawns have the tanks while Torch spawns don't?

This isn't a huge point of contention, but if it can be optimized before merging it'd be ideal.

Reply:
I am not entirely sure if it's worth adding a lot of checks for the type of the spawn only to make sure one spawns with a tank and the other doesnt.
I honestly think it's better to just remove it entirely and make atmosphere on Vox off-ship pure nitrogen.

Reply:
I don't believe the Vox Offship is supposed to start decompressed, but it most certainly has on me in the past. I'm in favor of keeping the tanks for that reason, but also don't see it as a major issue considering the state of the offship in general.